zerver/webhooks/github/view.py

```diff
@@ -160,18 +160,26 @@ def get_issue_body(helper: Helper) -> str:
     action = payload["action"].tame(check_string)
     issue = payload["issue"]
     has_assignee = "assignee" in payload
-    return get_issue_event_message(
+    base_message = get_issue_event_message(
         user_name=get_sender_name(payload),
         action=action,
         url=issue["html_url"].tame(check_string),
         number=issue["number"].tame(check_int),
         message=None
         if action in ("assigned", "unassigned")
         else issue["body"].tame(check_none_or(check_string)),
-        assignee=payload["assignee"]["login"].tame(check_string) if has_assignee else None,
         title=issue["title"].tame(check_string) if include_title else None,
     )
 
+    if has_assignee:
+        stringified_assignee = payload["assignee"]["login"].tame(check_string)
+        if action == "assigned":
+            return f"{base_message[:-1]} to {stringified_assignee}."
+        elif action == "unassigned":
+            return base_message.replace("unassigned", f"unassigned {stringified_assignee} from")
```

Comment:
I think ideally, we'd adjust the functions like `get_pull_request_event_message` and `get_issue_event_message` to use this nicer format; but that can potentially be follow-up work. @Joelute FYI since you've been working in this general area and might have thoughts on how to do that nicely.

Reply:
(In general, this kind of replacement stuff feels very messy and we should probably work to remove it, but I think we can do that in a follow-up)